src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -518,33 +518,37 @@ public static int findRepeatedSubstring(byte[] bases) {
      * @return                       Number of repetitions (0 if testString is not a concatenation of n repeatUnit's
      */
     public static int findNumberOfRepetitions(byte[] repeatUnit, byte[] testString, boolean lookForward) {
-        int numRepeats = 0;
+        final int repeatLength = repeatUnit.length;
+        final int testLength = testString.length;
+        final int lengthDifference = testLength - repeatLength;
```

Comment:
already documented `@return  Number of repetitions (0 if testString is not a concatenation of n repeatUnit's)`

I added tests for it